packages/web-monetization-demo/README.md

```diff
@@ -8,31 +8,38 @@ This also shows how a "verifier" can proxy SPSP requests to a "receiver" by
 adding extra headers to support [receipts](https://github.com/interledger/webmonetization.org/pull/51) (proof of payment).
 
 This package includes a minimal toy verifier for illustrative purposes, but for production use, see:
-[Receipt Verifier](https://github.com/coilhq/receipt-verifier)
+https://webmonetization.org/docs/receipt-verifier/
 
 ### Running the demo servers and client
 
-In 3 different terminals:
-
-To use the bundled toy verifier:
+In 4 different terminals:
 
 ```
 yarn client:dev
 yarn receiver:dev
+yarn server:dev
 yarn verifier:dev
```

Comment:
Having worked with this for a bit now, it would be great if we could add some information on all the environment variables one can specify, for all 4 of those.

Reply:
| Component      | Env Name           | Type       | Default | Comment |
| -------------- | :--                | -:         |  -:     |  -: 
| Verifier       | VERIFIER_PORT      | Number     | 4001 | | 
| Receiver       | RECEIVER_PORT      | Number     | 4000 | |
| Receiver       | BTP_PORT           | Number     | 3000 | set BTP_ENDPOINT when building extension |
| Server         | SERVER_PORT        | Number     | 4002 | |
| Server         | VERIFIER_URL       | String     | http://localhost:4001/verify | |

Yeah, maybe some kind of table/listing